packages/kbn-es-query/src/utils/__tests__/get_time_zone_from_settings.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import expect from '@kbn/expect';
+import moment from 'moment-timezone';
+import { getTimeZoneFromSettings } from '../get_time_zone_from_settings';
+
+describe('get timezone from settings', function () {
+  afterEach(function () {
+    moment.tz.setDefault();
+  });
+
+  it('should return the config timezone if the time zone is set', function () {
+    moment.tz.setDefault('America/Chicago');
```

Comment:
Looking at this again, I think we can actually get rid of all of the calls to `setDefault` in this entire file and these tests will still work as intended.